IBM/carbon-addons-iot-react
# (BarChartCard / TimeSeriesCard) domain range functionality

Closes #1338 

**Summary**

- Adds domain ranges to BarChartCard and TimeSeriesCard
- Fixes a BarChartCard bug where the chart was never able to be 'SIMPLE'

**Change List (commits, features, bugs, etc)**

- Updated to latest Carbon Charts version to pick up this bug fix (https://github.com/carbon-design-system/carbon-charts/pull/666)
- Changed the conditional logic for the different types of charts to support 'SIMPLE'
- Added new `domainRange` prop that accepts numbers (timestamps) or date instances


**Acceptance Test (how to verify the PR)**

- See https://deploy-preview-1339--carbon-addons-iot-react.netlify.app/?path=/story/watson-iot-timeseriescard--medium-single-line-domain-range
- See https://deploy-preview-1339--carbon-addons-iot-react.netlify.app/?path=/story/watson-iot-barchartcard--simple-bar-time-series

To see the domains actually change, the storybook will need to be run locally. I tried using knobs, but it wasn't forcing a rerender of the chart
